### DISCUSSION TITLE
How i get the value from editor ?

### DISCUSSION THREAD

**Discussion by helloindra** [outline/rich-markdown-editor#607] [Q&A] [answered]
Hello, i am trying to do this, but seems it doesnt work.

<```
Editor
          defaultValue={bodytext}
          placeholder="Hello world!"
          onChange={(e) => {
            setBodyText(e.target.value)
          }}
        />
```

<img width="748" alt="Screenshot 2021-12-20 164113" src="https://user-images.githubusercontent.com/75231253/146746345-0dbec4e1-e37e-4543-8241-9b19126038f8.png">

**Comment by tommoor** [comment, marked as answer]
Please see the documentation:

https://github.com/outline/rich-markdown-editor#onchange--value-void
